src/Cache/InMemoryDriver.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Tnapf\Config\Cache;
+
+use DateInterval;
+use Psr\SimpleCache\CacheInterface;
+use Tnapf\Config\Exceptions\InvalidCacheKeyException;
+
+class InMemoryDriver implements CacheInterface
+{
+    private array $storage = [];
+
+    public function get(string $key, mixed $default = null): mixed
+    {
+        return $this->has($key) ? $this->storage[$key] : $default;
```

Comment:
```suggestion
        return $this->storage[$key] ?? $default;
```

Reply:
I accidentally approved instead of requesting changes and need to post a comment to request changes now

Reply:
Accidentally approved instead of requesting changes and I dont wanna dismiss my review since Im not sure whether it removes the comments too

Reply:
The suggestion doesn't validate the incoming key, while `has()` does.